client/src/components/Survey.js

```diff
@@ -2,82 +2,88 @@ import React, {useEffect, useState} from "react";
 import {Button, Form, Select, RangeSlider, FormLayout, TextField} from '@shopify/polaris';
 
 export function Survey({ survey = {} }) {
-    const [answers, setAnswers] = useState({});
+    const [questions, setQuestions] = useState([]);
 
     useEffect(() => {
-        if(survey.questions) {
-            survey.questions.forEach((question) => {
-                answers[question.question] = null
-                setAnswers(answers)
-                console.log('useeffect')
-            })
-        }
-        // eslint-disable-next-line
+      let qs = survey.questions || []
```

Comment:
I'm not sure if you need this `useEffect`. I managed to make a similar page with client/src/components/QuestionList.js

Reply:
I don't know either!